web/components/input/buttons/ProgressGroup/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ProgressGroup = ({ count, onClick, activeIndex, lastValidIndex }) => {
+  const buttons = [];
+  for (let i = 0; i < count; i += 1) {
+    const isActive = (i === activeIndex);
+    buttons.push(
+      <button
+        key={i}
+        onClick={onClick}
```

Comment:
This API is definitely an improvement, but I'm still not digging https://github.com/nwhacks/nwhacks2019/pull/21/files#diff-35b1e54975e81f41e3665bf4708f4329R15 - where you have to get the page number by reading the text value of the element (`parseInt(e.currentTarget.textContent, 10) - 1`). We want to get the page number/index as the argument, nothing more. 

We could easily account for that by declaring another callback function like so:
`const func = () => onClick(i)`
and passing in this function as the onClick function:
`onClick={func}`

Reply:
> We want to get the page number/index as the argument, nothing more.

ah, was lazy. Good idea

Reply:
886cfcd